eval/src/vespa/eval/tensor/wrapped_simple_tensor.cpp

```diff
@@ -54,6 +54,17 @@ WrappedSimpleTensor::accept(TensorVisitor &visitor) const
     }
 }
 
+size_t
+WrappedSimpleTensor::count_memory_used() const
+{
+    size_t result = sizeof(WrappedSimpleTensor);
+    if (_space) {
+        result += sizeof(SimpleTensor);
+        result += _tensor.cells().size() * sizeof(SimpleTensor::Cell);
```

Comment:
this will greatly underestimate the actual memory used since most of the address will not be included. probably not a big deal.

Reply:
on second thought; the size of the Label struct multiplied by the number of dimensions in the tensor would be a good estimation for space used for the address part.